src/index.js

```diff
@@ -14,6 +14,7 @@ const teamList = teams.length ? teams.map(team => Team(team)) : [];
 const createAttachment = (hasMessages, { owner, repo }) => {
   let message = EMPTY_MESSAGE(owner, repo);
   let attachments = {};
```

Comment:
First time looking at this code, so not really familiar with how attachments are handled.
Is`attachments` ever an object? 

it looks like everywhere else you're using `attachments` or `subChannelAttachments` you're storing as an array.

Reply:
Yeah, this must have been an oversight. Nice catch 👍